pkg/apis/kops/cluster.go

```diff
@@ -695,11 +695,13 @@ type DNSControllerGossipConfig struct {
 }
 
 type RollingUpdate struct {
+	// DrainAndTerminate enables draining and terminating nodes during rolling updates.
+	// Defaults to true.
+	DrainAndTerminate *bool `json:"enabled,omitempty"`
```

Comment:
Looks like the json value needs updating, but I like the new name!

Reply:
Fixed in subsequent commit.